content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -718,11 +718,12 @@ crontab "my-new-cron-object" created
 
 ## Validation rules
 
-{{< feature-state state="alpha" for_k8s_version="v1.23" >}}
+{{< feature-state state="beta" for_k8s_version="v1.25" >}}
 
-Validation rules are in alpha since 1.23 and validate custom resources when the
+Validation rules are in beta since 1.25 and the
 `CustomResourceValidationExpressions` [feature
-gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled.
+gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled by default to validate custom resource
+ based on given validation rules. The feature could be disabled by turning off the feature gate `CustomResourceValidationExpressions`.
```

Comment:
Is this right?
```suggestion
based on _validation rules_. You can disable this feature by explicitly setting the
`CustomResourceValidationExpressions` feature gate to `false`,
for the `kube-apiserver` component.
```

Reply:
I like sftim's suggestion here
Maybe link to the kube-apiserver page that lists feature gates?
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/